### ISSUE TITLE
vue/key-spacing not working for code inside <template></template>

### ISSUE THREAD

**Issue by LPKnight** [vuejs/eslint-plugin-vue#1356]
Hi, the vue/key-spacing rule don't seem to lint code inside <template></template> in a .vue file. Code inside <template></template> does get checked and the rules work.

**Tell us about your environment**

- ESLint version: 7.13.0

- eslint-plugin-vue version: 7.1.0

- Node version: 13.6.0

**Please show your full configuration:**
module.exports = {
    root: true,
    parser: 'vue-eslint-parser',
    parserOptions: {
    'parser': 'babel-eslint',
    'sourceType': 'module'
    },
    env: {
    browser: true
    },
    extends: [
      'standard',
      'plugin:vue/strongly-recommended'
    ],
    plugins: [
      'standard',
      'promise',
      'vue'
    ],
    'rules': {
      'vue/v-bind-style': 2,
      'vue/key-spacing': [2, {beforeColon: true, afterColon: false}],
      'vue/singleline-html-element-content-newline': 0,
      'vue/max-attributes-per-line': 0,
      'no-debugger': process.env.NODE_ENV === 'production' ? 2 : 0
    }
 }

**What did you do?**
```
<template>
  <div :class="spec_cls" style="display:   none">
  </div>
</template>
```

**What did you expect to happen?**
There are some spaces between ‘display’ and 'none', Linter should report vue/key-spacing errors.
 btw:  which rule can detect the space between colon(v-bind ) and attribute in coding ，not until Compilation step.

**What actually happened?**
Linter finished without showing errors

Any help greatly appreciated. Thanks for help!!!

**Comment by ota-meshi** [comment]
Thank you for this issue.

The style attribute is CSS, not JavaScript, so it will not be parsed by eslint-plugin-vue. So I close this issue.

You may be able to use stylelint.
